src/portable/nuvoton/nuc121/dcd_nuc121.c

```diff
@@ -77,8 +77,11 @@ static bool active_ep0_xfer;
 /* RAM table needed to track ongoing transfers performed by dcd_edpt_xfer(), dcd_in_xfer(), and the ISR */
 static struct xfer_ctl_t
 {
-  uint8_t *data_ptr;         /* collectively, data_ptr and remaining_bytes track progress of endpoint transfers */
-  uint16_t remaining_bytes;
+  uint8_t *data_ptr;         /* data_ptr tracks where to next copy data to (for OUT) or from (for IN) */
+  union {
+    uint16_t in_remaining_bytes; /* for IN endpoints, we track how many bytes are left to transfer */

```

Comment:
thanks for the PR, normally I just used the **xferred_bytes**, as long as you feel good. Any names is OK :)